Airwallex/Card/AWXCardProvider.m

```diff
@@ -68,6 +69,19 @@ - (void)confirmPaymentIntentWithPaymentMethod:(AWXPaymentMethod *)paymentMethod
                                        }];
 }
 
+- (void)createPaymentConsentAndConfirmIntentWithPaymentMethod:(AWXPaymentMethod *)paymentMethod {
```

Comment:
This and the above method can be potentially merged with a shared completion handler. Not a big issue though

Reply:
Good idea, I just merged it 👍